src/EnrollCallAndSmsController.js

```diff
@@ -114,28 +115,34 @@ function (Okta, FormController, Footer, PhoneTextBox, TextBox, CountryUtil, Form
             profileData['phoneExtension'] = phoneExtension;
           }
 
-          if (isMfaEnroll) {
-            var factor = _.findWhere(transaction.factors, {
+          if(self.get('skipPhoneValidation')){
+            profileData['validatePhone'] = false;
+          }
+
+          var doEnroll = function (trans) {
+            var factor = _.findWhere(trans.factors, {
               factorType: self.get('factorType'),
               provider: 'OKTA'
             });
             return factor.enroll({
               profile: profileData
+            })
+            .fail(function (error) {
+              if(error.errorCode === 'E0000098') { // E0000098: "This phone number is invalid."
+                error.xhr.responseJSON.errorSummary += ' If you think the number is valid, please try again.';
```

Comment:
just a reminder to use the localized message once the i18n PR is verified by Vince and merged.